Makefile

```diff
@@ -73,3 +73,15 @@ run-ui-dev:
 # Builds the user interface on your local machine.
 build-ui:
 	cd ui; npm run build && npm run export; cd ..
+	
+#Incorporating Make docs commands from the Docs Makefile	
+jekyll=bundle exec jekyll
+
+site:
+	$(jekyll) serve --drafts --livereload
+
+build:
+	$(jekyll) build --drafts --livereload
+
+docker:
```

Comment:
@raghavaggarwal99 we'll have a conflict of `docker` in line 9. We may need to change line 86 to `docker-docs`